cmd/internal/cli/pull.go

```diff
@@ -91,7 +91,11 @@ func pullRun(cmd *cobra.Command, args []string) {
 	if PullImageName == "" {
 		name = args[0]
```

Comment:
would really like to rename this to `dstFileName` as the name saved in the cache dir can be different from the file name saved by the user